### PR TITLE
fixing earth engine cross-project access issue

### PR DESCRIPTION
**Pull request by kazob1998**
addressing issue: https://github.com/tnc-br/bugbash/issues/73

### CODE REVIEW

**Review by benwulfe — CHANGES_REQUESTED** [review]
how will this work for prod if you only add folks to the -test alias?
It should add to the correct group based on the project, yes?

**Comment by kazob1998** [comment]
-test is added for main branch ... for prd branch we keep old one

File: fraud-detection-update-sample/fraud_detection_process_sample.py [hunk 1]
```diff
@@ -144,13 +162,52 @@ def evaluate(self):
         is_invalid = combined_p_value <= self.p_value_theshold
         return is_invalid, combined_p_value, p_value_oxygen, o_mean, o_var, p_value_carbon, p_value_nitrogen
 
-def fraud_detection_process_sample(doc: dict):
+        _, p_value_oxygen = scipy.stats.ttest_ind_from_stats(
```
Comment (benwulfe) [inline fraud-detection-update-sample/fraud_detection_process_sample.py:165]: why is this code added here?  it wont run right?  also all three elements have already been calcd above yes?

File: fraud-detection-new-isoscape/origin_validation.py [hunk 2]
```diff
@@ -0,0 +1,128 @@
+# Import Libraries
```
Comment (benwulfe) [inline fraud-detection-new-isoscape/origin_validation.py:1]: why do you need a new file here for new-isoscape?  is it busted?  I thought new-isoscape was calling the other function so you dont need this.

**Review by benwulfe — CHANGES_REQUESTED** [review]
can you indicate how you tested the changes to all functions?  did you deploy them to the test project already or run each locally?

File: fraud-detection-new-isoscape/origin_validation.py (hunk 2 above)
Comment (kazob1998) [reply fraud-detection-new-isoscape/origin_validation.py:1]: yes that's true, File will be deleted in the coming commit

File: fraud-detection-update-sample/fraud_detection_process_sample.py (hunk 1 above)
Comment (kazob1998) [reply fraud-detection-update-sample/fraud_detection_process_sample.py:165]: yes , I will fix that in next commit

**Review by benwulfe — APPROVED** [review]
Please make sure this is tested once it lands in riversky backend.

**Pull request merged**